aptos-move/framework/aptos-stdlib/doc/math128.md

```diff
@@ -10,12 +10,14 @@ Standard math utilities missing in the Move Language.
 -  [Function `max`](#0x1_math128_max)
 -  [Function `min`](#0x1_math128_min)
 -  [Function `average`](#0x1_math128_average)
+-  [Function `mul_div`](#0x1_math128_mul_div)
```

Comment:
These doc changes are unrelated to the refactoring, right?

Reply:
No actually not. All inline functions have become visible in the model, because the models function tables are now derived from the parsed AST, and not the bytecode. In the bytecode, inlined functions are gone, so there were not visible in the model before (and hence docgen).

Reply:
So basically -- and unintentionally -- this PRC fixes a known incompleteness of docgen. It similar fixes some link decoration because the parsing of addresses had bugs which are fixed with this PR.

All those changes are _not_ piggybacked on this PR but a consequence of it.